app/app.js

```diff
@@ -161,7 +164,9 @@ process.on('SIGINT', shutdown);
 function shutdown() {
   log.info('Received kill signal. Shutting down...');
   state.shutdown = true;
-  QueueConnection.close();
+  dataConnection.close();
+  queueConnection.close();
+
```

Comment:
Was this working with the capital Q before?

Reply:
It was never actually running that code when it was a static function. Never caught that until now.